### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by darthmooguy**
Add warning for Windows for ARM users.

### CODE REVIEW

**Comment by darthmooguy** [comment]
Happened to me today, I would have liked to see this in the readme, not in the issues after the fact.

File: README.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ You can find the latest stable version here:
 
 [![GitHub All Releases](https://img.shields.io/github/downloads/Open-Shell/Open-Shell-Menu/total?style=for-the-badge&color=4bc2ee&logo=github)](https://github.com/Open-Shell/Open-Shell-Menu/releases/latest)
 
+### ⚠️ WARNING FOR WINDOWS ARM USERS ⚠️
```
Comment (ge0rdi) [inline README.md:26]: What about just `Warning for Windows for ARM users`?
Without the rather annoying all caps?
I guess those exclamation icons are enough to draw the attention.

Comment (ge0rdi) [reply README.md:26]: Or maybe just `Windows for ARM compatibility` ?

Comment (darthmooguy) [reply README.md:26]: Sure, I can change the title to be  less intense! It bricked my VM so I was a bit agitated at the time 😛

**Comment by Itsamejonah** [comment]
almost no one uses windows arm, but only on mobile

**Comment by Neustradamus** [comment]
It is linked to:
- https://github.com/Open-Shell/Open-Shell-Menu/issues/553
- https://github.com/Open-Shell/Open-Shell-Menu/pull/641
- https://github.com/Open-Shell/Open-Shell-Menu/pull/1912

**Comment by XenHat** [comment]
> almost no one uses windows arm, but only on mobile

Thank you for your feedback. We'll look into it.

File: README.md (hunk 1 above)
Comment (XenHat) [reply README.md]: We should add a few checks for the installer to only work on tested environments. There's been a few reports of the installer doing bad things to recent versions of windows.

**Comment by Neustradamus** [comment]
@ge0rdi: Good job! Thanks for the merging of @darthmooguy PR!

**Pull request merged**